docs/getting-started/README.md

```diff
@@ -171,11 +174,40 @@ Inside `scripts/` you will find `sample-script.js` with the following code:
 Run it with `npx buidler run scripts/sample-script.js`:
 
 ```
-$ npx buidler run scripts/deploy.js
+$ npx buidler run scripts/sample-script.js
 All contracts have already been compiled, skipping compilation.
+Deploying a Greeter with greeting: Hello, Buidler!
 Greeter deployed to: 0x7c2C195CD6D34B8F845992d380aADB2730bB9C6F
 ```
 
+### Using Buidler EVM as a node
+Buidler EVM can be run as a server or testing node. To do this, you just need to run
```

Comment:
Buidler EVM can run in a standalone fashion so that external clients can connect to it. This could be MetaMask, your Dapp front-end, or a script. To run Buidler EVM in this way, run:

Reply:
I copy-pasted that from here: https://buidler.dev/buidler-evm/#connecting-to-buidler-evm-from-wallets-and-other-software

Should I change it on both sections?

Reply:
sure